clients/drcachesim/tracer/instr_counter.cpp

```diff
@@ -121,6 +121,8 @@ hit_instr_count_threshold(app_pc next_pc)
         dr_mutex_unlock(mutex);
         return;
     }
+    auto timestamp = instru_t::get_timestamp();
+    instru_t::set_attached_timestamp(timestamp);
```

Comment:
Hmm, this might be confusing: someone might expect "attached_timestamp" to be from the single attach, not to be reset on every window start.  I think it might be best to have two separate timestamps.

Reply:
Ack. Added a new `retrace_attached_timestamp` and kept the original one as it was.